fedora/31/Dockerfile

```diff
@@ -4,6 +4,12 @@ MAINTAINER Alexey Kopytov <akopytov@gmail.com>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# Update repositories to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN echo "Remove disabled 'failovermethod' since Fedora 29"
+RUN sed -i '/^failovermethod=/d' /etc/yum.repos.d/*.repo
```

Comment:
There is no such entries since fedora:31 docker image.

Reply:
Ok, removed.